### DISCUSSION TITLE
JSON in etl

### DISCUSSION THREAD

**Discussion by nische** [ETLCPP/etl#590] [Ideas]
Hi, 
Are you currently planning to include JSON or similar protocols as part of etl? 

From my point of view this could be a nice feature, because out there are many library in cpp with lot of functions or libs in c for the better performance. But etl normally connect both of this. 

BR, 
Nico

**Comment by jwellbelove** [comment]
I don't have any immediate plans to add things like JSON. I am trying not to make the ETL into a 'do everything' library, (1) because I think libraries should have a focussed use-case, and (2) I don't have the time or the funds to work on this as a full time job. I earn around $40/month in sponsorship from users of the ETL, and that's not quite enough to give up the day job.

**Reply by nische** [reply]
Thanks for the reply. 
Yes you are right. a lib for parsing a standard protocol should be a own lib.
